proto/dwtools/abase/l1.test/Str.test.s

```diff
@@ -8835,6 +8836,80 @@ ghij
 
 //
 
+function strLinesNearestReport( test )
+{
+  let got, expected
+  test.case = 'charsRange[0] one letter';
+  got = _.strLinesNearestReport({ src : 'function add( x,y ) { return x + y }', charsRangeLeft : [ 1, 7 ], gray : 1, numberOfLines : 1 });
+  // console.log( got );
+  expected = 'f';
+  test.identical( got.nearest[ 0 ], expected );
```

Comment:
Please, check full container or each element of it. 
This test check says nothing about length of `got.nearest` and its content.
```js
  test.identical( got.nearest[ 0 ], expected1 );
  test.identical( got.nearest[ 1 ], expected2 );
  test.identical( got.nearest[ 2 ], expected3 );
```
Alternativelly:
```js 
  var exp = [ 'f', 'unction', '...' ]
  test.identical( got.nearest, expected );
```
If strings are not too long, use them inplace 
```js
  test.identical( got.nearest[ 0 ],  'f' );
  test.identical( got.nearest[ 1 ],  'unction', );
  test.identical( got.nearest[ 2 ], '...' );
```